src/_common.scss

```diff
@@ -22,3 +16,84 @@ $avatar-color-8: #b800b2;
 $avatar-color-9: #f22c44;
 
 $tablet: '(min-width: 768px)';
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Lato Regular'), local('Lato-Regular'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Regular.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Regular.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: bold;
+    src: local('Lato Bold'), local('Lato-Bold'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Bold.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Bold.woff') format('woff');
+}
+
+.ba,
+.ba-annotation-dialog,
```

Comment:
Are these needed only for the style guide? Seems like they will affect annotations in general.

Reply:
Most of this CSS will be needed by the annotations UI as a whole. I will clean it up as we add more components as well.

Reply:
`.ba` is only applied in the style guide right now. It will later be applied to annotations UI as well